cmd/utils.go

```diff
@@ -545,6 +576,12 @@ func getStackLabels(config *RootCommandConfig) (map[string]string, error) {
 			labelsMap := containerConfig["Labels"].(map[string]interface{})
 
 			for key, value := range labelsMap {
+				key = strings.Replace(key, "org.opencontainers.image", "dev.appsody.stack", -1)
```

Comment:
I think this transformation should happen when you are applying the `appsody build` labels rather than in the `getStackLabels()` func. Because other users of this func may expect the "org.opencontainers...." labels on the stack.

Reply:
True - have changed it